projects/commudle-admin/src/assets/styles/variables.scss

```diff
@@ -3,9 +3,6 @@ $max-page-width: 1440px;
 $tab-width: 768px;
 $mobile-width: 576px;
 
-// Fonts
-$primary-font-family: "Work Sans";
-
```

Comment:
I had to re-add it because it was still getting used at some places :)

Reply:
Sure!